docker-compose.yml

```diff
@@ -1,28 +1,23 @@
-version: "3.9"
-
-services:
-  flask_app:
-    container_name: flask_app
-    image: francescoxx/flask_live_app:1.0.0
-    build: .
-    ports:
-      - "5000:5000"
-    environment:
-      - DB_URL=postgresql://postgres:Favour98@flask_db:5432/kittie
-    depends_on:
-      - flask_db
-  flask_db:
-    container_name: flask_db
-    image: postgres:12
-    ports:
-      - "5432:5432"
-    environment:
-      - POSTGRES_PASSWORD=Favour98
-      - POSTGRES_USER=postgres
-      - POSTGRES_DB=kittie
-    volumes:
-      - pgdata:/var/lib/postgresql/data
-
-volumes:
-  pgdata: {}
-    
+version: '3'
+
```

Comment:
You should define your volume storage local path before mounting and using it like this
```yaml
volumes:
  postgres_data: {}


volumes:
      - postgres_data:/var/lib/postgresql/data
```

That way it works on any machine like Linux, MacOS and windows

Reply:
Here is a sample to reference: https://github.com/andela-sjames/jotpad/blob/main/docker-compose.yml